createItem.php

```diff
@@ -149,19 +149,16 @@ function validateForm() {
             ?>
 
             <label for="itemType">Item Type:</label>
-            <select id="itemType" name="type" required>
-                
-            </select>
+            
+            <input type="text" id="itemType" name="type" placeholder="Enter Item Type" required>
             <?php
                 if (!empty($itemType_err)) {
                     echo "<p id=\"itemTypeError\"style=\"color: red;\">$itemType_err</p>";
                 }
             ?>
 
             <label for="item">Item:</label>
-            <select id="item" name="item" required>
-                
-            </select>
+           <input type="text" id="item" name="item" placeholder="Enter Item Type" required>
```

Comment:
There is a pre-populated list on the db that we use to set item type

Reply:
The list is empty until you select a category

Reply:
When you select a category, it retrieves the item types under that category, like a dynamic list

Reply:
![image](https://github.com/yemoski/G-United/assets/87030215/f93232f6-1bfd-4b2c-a60c-a1b4caf3cc2b)